mod/metrics/set/type_plus_right/metric_value/checked_by.rb

```diff
@@ -19,19 +20,32 @@ def other_user_requested_check?
 end
 
 def checked?
-  !check_requested? && checkers.present?
+  checkers.present?
 end
 
 def check_requested?
-  items.first == "request" && items.size <= 2
+  items.first == "request"
```

Comment:
confused.  it looks to me like you were moving away from this representation in the migration above.  aren't we representing this with `check_requested_by`?